csep/utils/plots.py

```diff
@@ -568,14 +569,21 @@ def plot_basemap(basemap, extent, ax=None,  coastline=True, borders=False, linec
          linecolor (str): Color of borders and coast lines. default 'black',
          grid (bool): Draws a grid in the basemap
          grid_labels (bool): Annotate grid values
+         apprx (bool): If true, approximates transformation by setting aspect ratio of axes based on middle latitude
+         central_latitude (float): average latitude from plotting region
 
      Returns:
          :class:`matplotlib.pyplot.ax` object
 
      """
     if ax is None:
         fig = pyplot.figure()
-        ax = fig.add_subplot(111, projection=ccrs.PlateCarree(central_longitude=0.0))
+        ax = fig.add_subplot(111, projection=projection)
+        if apprx:
```

Comment:
maybe add `& projection == ccrs.PlateCarree()`? (not needed if you are sure that `plot_basemap()` will never be called by a user)

Reply:
the default arg is projection=ccrs.PlateCarree. I believe its good as Bill left it, to accept different projections. Also the default platecarree looks good, for quick plotting without worrying about projections.

Reply:
i think `plot_basemap` will probably be called by other users especially if they are trying to plot catalogs on top of forecasts or other data sets. is there a downside to letting users choose projections in the `plot_basemap` function?

Reply:
What I meant is: the 'fast'/apprx "projection" works by rescaling the axes. This rescaling is incompatible with any other "true" (non-PlateCaree) projection, which should be caught by adding `& projection == ccrs.PlateCarree()` to the if statement.

Reply:
thats a good point. in the other functions i handled that case by forcing `projection=ccrs.PlateCarree()`. for example, in `plot_catalog()`. not sure why i didn't do that in basemap also, because if the user chooses the *fast* option, we should force PlateCarree(). 

now that im thinking about it, there are some other parameters that might lose continuity between these cartopy enhanced calls, eg, `extent`